python/test/t_PiecewiseLinearDistribution_std.py

```
@@ -0,0 +1,504 @@
+buf.seek(0)
+dist_loaded = pickle.load(buf)
+ott.assert_almost_equal(dist_loaded.getX(), distribution.getX())
+ott.assert_almost_equal(dist_loaded.getY(), distribution.getY())
+ott.assert_almost_equal(dist_loaded.computePDF(1.0), distribution.computePDF(1.0))
+ott.assert_almost_equal(dist_loaded.computeCDF(1.0), distribution.computeCDF(1.0))
+print("Pickle save/load OK")
+
+# ---------------------------------------------------------------------------
+# 18. Serialization via Study (XML)
+# ---------------------------------------------------------------------------
+tmpfile = tempfile.mktemp(suffix=".xml")
```

Ruff (0.16.3)

[error] Use of insecure and deprecated function (`mktemp`)

(S306)